app/CMakeLists.txt

```diff
@@ -15,4 +15,12 @@ if (APPLE)
     target_sources(DeltaPatcher PRIVATE ${bundle_icon})
     set_source_files_properties(${bundle_icon} PROPERTIES MACOSX_PACKAGE_LOCATION Resources/)
     set_target_properties(DeltaPatcher PROPERTIES MACOSX_BUNDLE TRUE MACOSX_BUNDLE_INFO_PLIST ${CMAKE_CURRENT_SOURCE_DIR}/mac_stuff/Info.plist)
-endif()
\ No newline at end of file
+endif()
+
+install(TARGETS DeltaPatcher DESTINATION bin)
```

Comment:
the install part should be conditional to Linux-based systems. On Windows and MacOS share/icons, and .desktop files do not make much sense.